winlogtimeline/util/project.py

```diff
@@ -1,34 +1,44 @@
+import json
 import os
 import sqlite3 as sql
 from datetime import datetime
 
+import util
```

Comment:
This line:
`import util`
is throwing me an error.

Reply:
Relative imports are funky. Fixed it along with a few other things in this next change.